lib/kitchen/directions/bake_notes.rb

```diff
@@ -4,7 +4,7 @@ module Kitchen
   module Directions
     module BakeNotes
       def self.v1(book:)
-        book.notes.each do |note|
+        book.notes('$:not(.checkpoint):not(.theorem)').each do |note|
```

Comment:
The filtered classes `checkpoint` and `theorem` may be calculus specific. I think we're going to need a larger-scale notes refactor -- I'll probably do that in another PR shortly.